Makefile

```diff
@@ -1,5 +1,18 @@
+# Some important environment variables
 EMULATOR ?= simh
 
+# Sometimes you _really_ need to use a different `touch` or `rm`.
+TOUCH ?= touch
+MKDIR ?= mkdir
```

Comment:
This is always used with `-p`. autoconf usually does this as `MKDIR_P ?= mkdir -p`, since then you can replace it with a script that does the same thing (like `mkinstalldirs`) if you have a non-POSIX `mkdir`.

Reply:
Updated to include -p.  Removed -p from all users.